control_msgs/action/AntipodalGripperCommand.action

```diff
@@ -0,0 +1,11 @@
+sensor_msgs/JointState command # The velocity and effort fields are optionally and not used if empty.
+---
+float64 position  # The current gripper gap size (in meters)
+float64 effort    # The current effort exerted (in Newtons)
```

Comment:
Suggest updating the feedback to also be a `JointState`

Reply:
Done